HowTo_Execute.txt

```diff
@@ -0,0 +1,25 @@
+(WIP) User's Manual
```

Comment:
Can you put all your HowTo's into a folder called "tutorials"?

Also, rename the tutorial text to use lowercase & be more descriptive : "how_to_pull_request" and "how_to_execute_cli"